core/site_scons/models/T3T1/emulator.py

```diff
@@ -59,6 +59,26 @@ def configure(
         sources += ["embed/trezorhal/unix/optiga.c"]
         features_available.append("optiga")
 
+    if "tropic" in features_wanted:
+        sources += [
+            "vendor/libtropic/src/libtropic.c",
+            "vendor/libtropic/src/lt_crc16.c",
+            "vendor/libtropic/src/lt_hkdf.c",
+            "vendor/libtropic/src/lt_l1.c",
+            "vendor/libtropic/src/lt_l1_port_wrap.c",
+            "vendor/libtropic/src/lt_l2.c",
+            "vendor/libtropic/src/lt_l2_frame_check.c",
+            "vendor/libtropic/src/lt_l3.c",
+            "vendor/libtropic/src/lt_random.c",
+            "vendor/libtropic/hal//port/unix/lt_port_unix.c",
+            "vendor/libtropic/hal/crypto/trezor_crypto/lt_crypto_trezor_aesgcm.c",
+            "vendor/libtropic/hal/crypto/trezor_crypto/lt_crypto_trezor_ed25519.c",
+            "vendor/libtropic/hal/crypto/trezor_crypto/lt_crypto_trezor_sha256.c",
+            "vendor/libtropic/hal/crypto/trezor_crypto/lt_crypto_trezor_x25519.c",
+        ]
+        defines += ["USE_TREZOR_CRYPTO"]
```

Comment:
Maybe this should be `USE_TS_CRYPTO`, no?

```suggestion
        defines += ["USE_TS_CRYPTO"]
```

Reply:
> Maybe this should be `USE_TS_CRYPTO`, no?

Actually, I was thinking *not*, because this is the emulator build, in which we would still use `TREZOR_CRYPTO`, because the TS chip is not there. What do you think, @pavelpolach321?

Reply:
In libtropic we use USE_TREZOR_CRYPTO define if we want to use **trezor_crypto** as a source of cryptographic code which is running on host (MCU) side. 

Example: https://github.com/tropicsquare/libtropic/blob/master/hal/crypto/trezor_crypto/ts_trezor_x25519.c#L1

Therefore I think that passing USE_TREZOR_CRYPTO to compilation of libtropic files will work in your case?